resources/assets/js/components/UpdateProject.js

```diff
@@ -0,0 +1,154 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import { Link } from 'react-router';
+import {browserHistory} from 'react-router';
+import MyGlobleSetting from './MyGlobleSetting';
+import $ from 'jquery';
+
+class UpdateProject extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {projectName: '', projectInformation: '', projectPhone: '', projectDob: '', projectPosition: 'intern', projectGender: '1', projectAvatar: null};
+    this.handleChangeName = this.handleChangeName.bind(this);
+    this.handleChangeInformation = this.handleChangeInformation.bind(this);
+    this.handleChangePhone = this.handleChangePhone.bind(this);
+    this.handleChangeDob = this.handleChangeDob.bind(this);
+    this.handleChangePosition = this.handleChangePosition.bind(this);
+    this.handleChangeGender = this.handleChangeGender.bind(this);
+    this.handleChangeAvatar = this.handleChangeAvatar.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount(){
+    axios.get(MyGlobleSetting.url + `/api/projects/${this.props.params.id}`)
+    .then(response => {
+      this.setState({ projectName: response.data.name, projectInformation: response.data.information, projectPhone: response.data.phone, projectDob: response.data.date_of_birth, projectPosition: response.data.position, projectGender: response.data.gender, projectAvatar: response.data.avatar });
+    })
+    .catch(function (error) {
+      console.log(error);
```

Comment:
Maybe it is better to send this error message to the error console: `console.error(error)`